resources/qml/pages/ContactsPage.qml

```diff
@@ -61,7 +61,7 @@ Page {
 
                 Label {
                     id: nameId;
-                    text: name
+                    text: qsTr("name")
```

Comment:
This is an id which gets replaced by c++ code with the contact name. Not translateable.